data/test/news/articles/c0000000003o.json

```diff
@@ -88,39 +85,29 @@
             "locator": "urn:bbc:pips:pid:p067gq6h",
             "blocks": [
               {
-                "type": "rawVideo",
-                "model": {
-                  "guidance": null,
-                  "isLive": false,
-                  "duration": "PT1M34S",
-                  "allowOffsiteEmbedding": false,
-                  "locator": "urn:bbc:pips:pid:p067gq6h"
-                }
-              },
-              {
-                "type": "altText",
-                "model": {
-                  "blocks": [
+                "type" : "aresMedia",
+                "model" : {
+                  "id" : "p067gq6h",
+                  "subType" : "clip",
+                  "format" : "audio_video",
+                  "title" : "Mark and Isla Royal Wedding",
+                  "synopses" : {
+                    "short" : "Stop the press! It’s a Mark and Isla Royal Wedding special…",
+                    "medium" : null,
+                    "long" : null
+                  },
+                  "imageUrl" : "ichef.bbci.co.uk/images/ic/$recipe/p067grwg.jpg",
+                  "embedding" : true,
+                  "warnings" : { },
```

Comment:
For linting this should be `{}`. I'm also questioning whether it's worth having as an empty object? As this is plural is it meant to be an array? Or is this going to be an object containing key value pairs? 
Clarification on the contents of `warnings` would help

Reply:
Example warnings:
`"warnings": {
        "short": "Contains very strong language and some sexual content.",
        "long": "Contains very strong language and some scenes of a sexual nature."
      }`

Reply:
we should reflect long and short in the `schema.yaml`

Reply:
Actually there is a lot more we need to reflect in the `schema.yaml` which should be handled in the issue https://github.com/BBC-News/simorgh/issues/766